pkg/services/store/entity/migrations/entity_store_mig.go

```diff
@@ -60,7 +61,7 @@ func initEntityTables(mg *migrator.Migrator) string {
 			{Name: "errors", Type: migrator.DB_Text, Nullable: true},   // JSON object
 		},
 		Indices: []*migrator.Index{
-			{Cols: []string{"namespace", "group", "resource", "name"}, Type: migrator.UniqueIndex},
+			{Cols: []string{"group", "resource", "namespace", "name"}, Type: migrator.UniqueIndex}, // == key
```

Comment:
keys are used in order in mysql at least, the index order was optimized for most queries being namespaced rather than to match url order